src/simple_idml/indesign/scripts/export.jsx

```diff
@@ -144,7 +144,7 @@ try {
                 try{
                     ignoreSpreadOverrides = false;
                 }
-                catch(e){}
+                catch(e){ alert("Ignoring exception setting property ignoreSpreadOverrides: " + e)}
```

Comment:
For the various messages, what do you think of something like : 
`alert('Warning: cannot set "ignoreSpreadOverrides" option. Error was: ' + e)` ?

Reply:
Yep, I like that. Will adjust.